index.html

```diff
@@ -360,6 +360,10 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
             are missing from |td|, [= exception/throw =] a
             {{"TypeError"}} and abort these steps.
           </li>
+          <li>
```

Comment:
This section needs to be before the previous step:
1. fill in the default values
2. check for mandatory values.
There might be a case that a mandatory value does have a default value.

Reply:
Looks like we cross-commented with @egekorkan here, and they are opposite. Ege, please check if my comment above is valid.

Reply:
I checked the TD spec and it seems there are no cases for mandatory properties that would have default values.
Therefore we can omit the filling up step, and also we can go with Ege's proposal.